src/snowflake/snowpark/_internal/ast.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+import base64
+import itertools
+import sys
+
```

Comment:
should the `proto.ast_pb2` file get checked in at all? I assume it's generated from `ast.proto`? Would maybe remove it from the commit and add acomment to run first ast-make.sh before using this code.

Reply:
The build process is TBD. I'm thinking we shouldn't even check in the proto file, since its master copy is in the main Snowpark repo.

Reply:
We arleady have a task for this. Resolving.